spec/spec_helper_acceptance.rb

```diff
@@ -14,8 +14,8 @@
   c.before :suite do
 
   hosts.each do |host|
-      scp_to(host, Dir.glob('pkg/puppetlabs-strings*.gz').first, 'strings.tar.gz')
-      on host, puppet('module', 'install', 'strings.tar.gz')
+      scp_to(host, Dir.glob('puppet-strings*.gem').first, 'puppet-strings.gem')
```

Comment:
Does this work correctly if you have a dirty working directory with an old gem in it?